scripts/update_python.py

```diff
@@ -10,15 +10,19 @@
 We only supply binaries for windows and macOS, but we do it very different ways for those two OSes.
```

Comment:
Perhaps add another sentence here:  "On linux, we simply depend on the system version of python".